package.json

```diff
@@ -28,6 +28,7 @@
     "Paul Wright <paul@paintedbicycle.com>"
   ],
   "scripts": {
+    "start": "npm run patternlab:serve",
```

Comment:
This was removed on purpose to allow those who want to use yarn or npm. Also since it's just a hook to serve I felt it was not needed.